tests/test_security_files.py

```diff
@@ -0,0 +1,7 @@
+def test_insecure_post_fails(flask_test_client):
```

Comment:
File name not correct - test_security_files isn't really what its doing.  Need a better more descriptive name than that "files" is the offending word I think